extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -144,6 +144,8 @@ export const scriptFriendlyName = localize('scriptFriendlyName', "Script");
 export const tableFriendlyName = localize('tableFriendlyName', "Table");
 export const viewFriendlyName = localize('viewFriendlyName', "View");
 export const storedProcedureFriendlyName = localize('storedProcedureFriendlyName', "Stored Procedure");
+export const preDeployScriptFriendlyName = localize('preDeployScriptFriendlyName', "PreDeployment Script");
```

Comment:
this is the default file name right? Would it also make sense to stick to the same default name as SSDT which uses "Script.PostDeployment1.sql"?